### DISCUSSION TITLE
Can this helper be used with gpg keys stored on a removeable device?

### DISCUSSION THREAD

**Discussion by doolio** [languitar/pass-git-helper#323] [Q&A] [answered]
I have my `gpg` keys on a removeable device. Typically when I query my `~/.password-store` I'm prompted to decrypt the file on this removeable device. Will this also be the case with this helper?

Edit: I'm not getting prompted on my hardware device as  usually do so just want to understand if it is possible or if further configuration is needed.

**Comment by languitar** [comment, marked as answer]
You need to configure a graphical prompt for the passphrase. Then things should work as usual. The prompting is done by gpg and pass-git-helper does not change anything about it apart from needing the stdin and stdout to communicate with git.

**Reply by doolio** [reply]
Hmm ok. With my particular device I get prompted on the device itself (and even then just to OK the decryption not to enter a passphrase) and not via a graphical prompt on my machine.

Anyway, thanks for the prompt reply.
